East Indies fixes for September

KrasnaMerah opened the pull request:
Fixing many portraits for the East Indies, and proper character pathing for Moluccas and Papua

KrasnaMerah committed:
Update 070924

Fixing many portraits for the East Indies, and proper character pathing for Moluccas and Papua

KrasnaMerah committed:
Sutomo Update

I missed to replace his advisor portrait, bruh....

marijn211 committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

KrasnaMerah committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

marijn211 committed:
Update MLK - Maluku.txt

marijn211 reviewed (dismissed):
[23:28:41][no_game_date][effectimplementation.cpp:5472]: events/indonesia.txt:979: add_country_leader_role: Invalid country leader trait cautious_japan
[23:28:41][no_game_date][effectimplementation.cpp:5472]: events/indonesia.txt:999: add_country_leader_role: Invalid country leader trait cautious_japan

KrasnaMerah committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked

KrasnaMerah committed:
Merge branch 'East-Indies-Reworked' of https://github.com/deliciousmods/1956_beta into East-Indies-Reworked

KrasnaMerah committed:
Indies October Update

Adding Van Mook's and Thamrin's portrait, adding an event to retire Maarten van Dulm

KrasnaMerah committed:
Merge branch 'Beta-Edits' into East-Indies-Reworked
